internal/characters/tighnari/skill.go

```diff
@@ -0,0 +1,54 @@
+package tighnari
+
+import (
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+)
+
+var skillFrames []int
+
+const skillHitmark = 20
+
+func init() {
+	skillFrames = frames.InitAbilSlice(15)
+}
+
+func (c *char) Skill(p map[string]int) action.ActionInfo {
+	ai := combat.AttackInfo{
+		ActorIndex: c.Index,
+		Abil:       "Vijnana-Phala Mine",
+		AttackTag:  combat.AttackTagElementalArt,
+		ICDTag:     combat.ICDTagNone,
+		ICDGroup:   combat.ICDGroupDefault,
+		StrikeType: combat.StrikeTypeDefault,
+		Element:    attributes.Dendro,
+		Durability: 25,
+		Mult:       skill[c.TalentLvlSkill()],
+	}
+
+	c.Core.QueueAttack(
+		ai,
+		combat.NewCircleHit(c.Core.Combat.Player(), 2, false, combat.TargettableEnemy),
+		skillHitmark,
+		skillHitmark,
+	)
+	c.Core.QueueParticle("tighnari", 3.5, attributes.Dendro, c.ParticleDelay)
```

Comment:
this should be done in the usual way of handing `count` to QueueParticle: set count to 3 and then do count++ if rand < 0.5

Reply:
also it should be skillHitmark+c.ParticleDelay